Library/Homebrew/livecheck/strategy/extract_plist.rb

```diff
@@ -107,6 +107,14 @@ def self.find_versions(cask:, url: nil, regex: nil, options: Options.new, &block
             cask_copy = Cask::CaskLoader.load(cask.sourcefile_path)
             cask_copy.allow_reassignment = true
             cask_copy.url url
+
+            cask.livecheck.options.to_h.each_key do |options_key|
```

Comment:
```suggestion
            cask.livecheck.options.url_options.each_key do |options_key|
```

This should work fine either way but using `Options.url_options` may save some work in the future when there are more `Options` values that aren't `url` options.